knimin/lib/tests/test_data_access.py

```diff
@@ -297,6 +297,99 @@ def test_get_ag_barcode_details(self):
             del(obs[key]['registered_on'])
         self.assertEqual(obs, exp)
 
+    def test_plate_operations(self):
```

Comment:
Understand your comment about being "dependent", but usually unit test test small functionality. I think you should split up the tests for each of them, and if you need to create another plate your one of your other tests, you can create it. It may seem some code duplication, but having unit tests that actually test units are really helpful to debug later.